transport/utils/custom_validations.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"fmt"
+
+	svcmodel "release-manager/service/model"
+
+	"github.com/go-playground/validator/v10"
+)
+
+func ValidateReleaseMessageType(fl validator.FieldLevel) bool {
+	return fl.Field().String() == svcmodel.ReleaseMessageTemplateType
+}
```

Comment:
Custom validation is added. I could add also do 

```
Type  *string `json:"type" validate:"required,eq=release_message"`
```

In this case, I couldn't pass the variable from the model and had to hard-code the value instead.